controllers/kubebuilder_rbac.go

```diff
@@ -1,26 +1,21 @@
 package controllers
 
+// LlamaStackDistribution CRD permissions
 //+kubebuilder:rbac:groups=llamastack.io,resources=llamastackdistributions,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=llamastack.io,resources=llamastackdistributions/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=llamastack.io,resources=llamastackdistributions/finalizers,verbs=update
 
-// +kubebuilder:rbac:groups="core",resources=serviceaccounts,verbs=get;list;watch;create;update;patch;delete
+// Deployment permissions - controller creates and manages deployments
+//+kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
 
-// +kubebuilder:rbac:groups="core",resources=pods/log,verbs=*
-// +kubebuilder:rbac:groups="core",resources=pods/exec,verbs=*
-// +kubebuilder:rbac:groups="core",resources=pods,verbs=*
+// Service permissions - controller creates and manages services
+//+kubebuilder:rbac:groups="",resources=services,verbs=get;list;watch;create;update;patch;delete
 
-// +kubebuilder:rbac:groups="core",resources=persistentvolumes,verbs=*
-// +kubebuilder:rbac:groups="core",resources=persistentvolumeclaims,verbs=*
+// PVC permissions - controller creates PVCs (immutable after creation, no update/patch needed)
+//+kubebuilder:rbac:groups="",resources=persistentvolumeclaims,verbs=get;list;watch;create
 
-// +kubebuilder:rbac:groups="apps",resources=deployments/finalizers,verbs=*
-// +kubebuilder:rbac:groups="core",resources=deployments,verbs=*
-// +kubebuilder:rbac:groups="apps",resources=deployments,verbs=*
+// ConfigMap permissions - controller reads user configmaps and creates operator config configmaps
+//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;create
```

Comment:
wont require "update" if https://github.com/llamastack/llama-stack-k8s-operator/blob/main/config/manager/distribution-configmap.yaml changes in new release?

Reply:
That's true! It would update instead of delete & re-create in that case.

I've updated the `configmaps` verbs to include `update` and `patch`.